security/KeePassXC/Portfile

```diff
@@ -164,6 +144,8 @@ post-destroot {
 
     ln -s ${applications_dir}/KeePassXC.app/Contents/MacOS/keepassxc-cli \
         ${destroot}${prefix}/bin/keepassxc-cli
+    ln -s ${applications_dir}/KeePassXC.app/Contents/MacOS/keepassxc-proxy \
+        ${destroot}${prefix}/bin/keepassxc-proxy
```

Comment:
Nit-picky observation, not important enough to be addressed as part of this PR:

Try to avoid using copy-pasta for paths, particularly when there are two or more references to the same location. Instead, create local variables, and reference those:

```
post-destroot {
    set kp_doc_path ${destroot}${prefix}/share/doc/${name}

    xinstall -d ${kp_doc_path}
    xinstall -W ${worksrcpath} COPYING LICENSE.BSD LICENSE.CC0 \
             LICENSE.GPL-2 LICENSE.GPL-3 LICENSE.LGPL-2.1 LICENSE.LGPL-3 \
             LICENSE.MIT LICENSE.NOKIA-LGPL-EXCEPTION LICENSE.OFL \
             ${kp_doc_path}

    set kp_app_path ${applications_dir}/KeePassXC.app/Contents/MacOS
    set kp_bin_path ${destroot}${prefix}/bin

    ln -s ${kp_app_path}/keepassxc-cli \
        ${kp_bin_path}/keepassxc-cli
    ln -s ${kp_app_path}/keepassxc-proxy \
        ${kp_bin_path}/keepassxc-proxy
```